### ISSUE TITLE
24.问与答

### ISSUE THREAD

**Issue by Pathwork-In-China** [Pathwork-In-China/Pathwork-Lectures#26]
24. Questions and Answers
### 问与答
Pathwork Guide Lecture No. 24  |  February 28, 1958
心路指导讲座 第24篇 | 1958.2.28

24.1  Greetings in the name of the Lord. I bring you blessings, my friends. Again I will answer your questions tonight, instead of holding a lecture. I will resume regular lectures after most of your questions have been answered. It is a very good sign if my little group has questions, for this is proof that your minds are active, producing, searching. This is always the first step to spiritual development, to an inner awakening. And therefore we consider it important to satisfy your queries as much as this is possible and postpone the regular lectures. However, I will point this out:  There are some questions that I will not answer as thoroughly as some of you may wish, for we can see that taking up time on these matters would not be of real importance for your development, at least at the present time. On questions that have real significance and importance we will spend more time. I also wish to inform you that I plan a new series of lectures after the present questions—or some new ones that may come up—have been taken care of. This series will represent a development course. For many friends here sincerely wish to advance spiritually and yet do not quite know how to begin, or what to do—all those friends who do not come for private sessions. It is important that they have a definite idea how to go about it in actual practice. And so we will be working together in that direction. Thus, the following series will deal with the systematic development of the spirit and the soul. It will help you to solve your psychological problems as well as can be done without a personal teacher. I will show you exactly how to go about it. In-between, I may give a single lecture on a particular subject, as I have done in the past. It may appear to you, at first, as though it had nothing to do with this development course, yet you will eventually find out that it does have a close bearing on it.
以主之名问候你们。我为你们带来祝福，我的朋友们。今晚我会再次回答你们的问题，而不是给出讲座。我会在你们的大多数问题得到回答后再恢复常规讲座。我的小组有问题要问是一个很好的迹象，因为这证明你们的头脑是活跃的、有产出的、善于搜索的。这总是通往精神发展、通往内心觉醒的第一步。因此我们认为尽可能地满足你们的提问是一件重要的事，并且延后了有规律的讲座。不过，我要指出的是：有些问题我不会做出你们希望中的那种详尽程度的回答，因为我们可以看到在那些问题上花费时间不会对你们的发展有真正的重要性，至少从目前来说如此。对那些有真正重要性的问题我们会花更多时间。同时我也想告诉你们，我计划在当前的问题——以及可能临时出现的新问题——得到处理后，会有新的一系列讲座。这个系列会代表一个发展课程。针对那些在座的真诚希望得到精神进步却又不太清楚如何开始，或者应该怎么做的朋友们——也就是说所有那些不参加私下会议的朋友们。让他们对于“实际行动中如何进行”有一个明确的概念是很重要的。所以我们会在那个方向上共同努力。因此，接下来的这一系列（讲座）会处理精神/心灵和灵魂的系统性发展问题。它会帮你在没有个人导师的情况下尽可能好地处理你的心理问题。我会向你们准确细致地展示如何进行。在这个过程中，我可能会就某个特定主题给出一次讲座，就像我之前做过的那样。一开始，它对你来说似乎跟这系列发展课程没有任何关联，不过你们最终会发现它确实对此有紧密的关联。

24.2  Before I turn to your questions, I would like to say that I recognize in this room a sad heart. Oh, there are a few people here with problems. There is a sadness here, a very specific sadness. Every type of sadness or sorrow can only exist because you human beings are ignorant of the perfection of the Creation, of the perfection of divine law. And whatever happens must be for the good. It must come out as a happy solution in the long run. For this is the way divine law is made. So wherever there is sadness or sorrow—or disharmony for that matter—it can only exist because of your ignorance. And the more you come here, the more you learn and especially the more you become a better person. The more you develop spiritually, the more the curtain will lift and you will realize that there is never any cause for sadness. There is nothing that can ever cut you off from God and from all of those you love, no matter what may happen now. Real truth, absolute truth, can only give reason for rejoicing. I know it is difficult for you to understand this. And if you can understand it at all, it is only in your intellect, where perhaps you can grasp this truth. But you will have to come to know this truth in your innermost being, so that you feel it through and through without any question in your mind, without any doubt. And if you have not achieved this goal yet, you know what remains for you to do. If sadness, or disappointment, or any lack of understanding comes your way, the reason is that you should use these events to get yourself a little further up on the ladder.
在我转向你们的问题之前，我想说我在这间屋子里看到一颗悲伤的心。哦，在座的有些人（的内心）是怀着困难的。这里有一种伤心的情绪，一种非常特定的伤心情绪。每一种哀伤或悲痛之所以能存在，仅仅是因为你们人类没有意识到宇宙造物的完美之处，以及神圣法则的完美之处。因此不论发生什么事情，背后一定有好的原因。从长远来看它一定是一种愉快的解决方式。因为这正是神圣法则的运作方式。所以只要有哀伤或悲痛（的情绪）存在——或者任何形式的不和谐——它只能因为你的无知而存在。你来这里越多，就学到越多，而且你就特别会越加成为一个更好的人。你在精神/心灵层面发展得越多，就有越多的帷幕被提起，而你会意识到自己从来都没有理由需要悲伤。不论现在发生什么，从来没有任何东西能使你与上帝和一切你爱的人和事分离开来。真正的真理，绝对的真理，只会给人理由去感到欢欣与喜悦。我知道要你们理解这一点有困难。而如果你们能理解的话，也只是在智力层面的理解，在这里你也许能领悟到这个事实。但你必须从自己内心最深处的地方认识这个事实，这样你才能彻头彻尾、贯穿始末的感受到它，而不带任何头脑的疑虑和内心的疑惑。如果你还没有达到这个目标，你就知道了自己还需要做的是什么。如果有悲伤、失望或者任何缺乏理解力的情绪发生在你身上，其中的原因是你应该利用这些事件使自己在梯子上更进一小步。

24.3  And now, my friends, I am ready for your questions.
现在，我的朋友们，我准备好回答你们的问题了。

24.4  QUESTION:  Is it all right if we ask first the questions on the “Pistis Sophia?”
问题：我们可以先问有关《救世主的智慧》的问题吗？

24.5  ANSWER:  Yes, I would suggest that you ask first the questions on the “Pistis Sophia” and then deal with the other general questions you have, as far as this is possible tonight.
回答：可以，我会建议今晚你们先问关于《救世主的智慧》的问题，然后再尽可能地处理其他的一般性问题。

24.6  QUESTION:  Is the scheme or skeleton of the “Pistis Sophia” about the spheres a factual or a Gnostic concept—the division of the different worlds—from the Ineffable to the Outer Darkness?  If not, it is no use going too deeply into it in the discussion.
问题：《救世主的智慧》中关于不同位面的体系和架构——不同世界的分隔——从不可言喻的到外部黑暗这是一种事实性的，换句话说也就是诺斯替（认知）性的概念吗？如果不是的话，对这部分内容进行太深度的讨论就没有意义了。

24.7  ANSWER:  Let me put it this way:  Roughly speaking, it is correct with minor deviations, with minor errors and omissions. I mean, there is a little more to it and some of it is a little different, but on the whole it is rather correct. But the names, of course, are not necessarily those that we use. For we, in the spirit world, have a language of pictures, as you know. We do not pay too much attention to the actual names, but in essence, in meaning, it is more or less correct. Nevertheless, I would like to say that at the present time it is not important that we go too deeply into this. Perhaps when you have finished, my friend, with your lectures on the “Pistis Sophia,” I will then give you at one time or another a little more information on this subject.
问题：让我这样说吧：粗略地说，它是正确的，但会有少数偏离、少数错误和遗漏。我的意思是，还有些情况是其中没提到的，有些提到的情况实际是不同的，但总体来说它是比较正确的。不过当然了，其中的命名不一定是我们使用的名称。因为我们，处于精神世界里，使用的是一种图像性的语言，这是你们知道的。我们不太关注实际名字，但从本质上，以及从含义上来说，它大体是正确的。尽管如此，我想说的是目前我们不太需要太深度地讨论这部分。也许当你完成了你的关于《救世主的智慧》的讲座时，我的朋友，那时我会再找时间给你一些额外的这方面信息。

24.8  QUESTION:  Why did Jesus need the reinvestiture of His robes of glory—there are three—before being able to speak in openness, face-to-face to the disciples?
问题：为什么耶稣首先需要对祂的荣耀长袍——有三件这样的长袍——进行再认证，然后才能与信徒们进行开放的、面对面的交谈？

24.9  ANSWER:  You all know, my friends, that spiritual objects, whether landscapes, clothes or whatever else—it always seems so unbelievable to human beings that all this should exist in spirit or in subtle matter—are only an expression of the state of mind or of the state of affairs. Such objects are not as you may think, merely symbols that express a meaning for you human beings, but is quite the opposite. Your objects, whatever you have here on this earth sphere, are nothing but symbols of what exists in the spirit world. In other words, it is the other way round. Now, due to the series of lectures I have given on the Fall and on Salvation, you will understand that Jesus, when He came to the earth sphere and afterward to the spheres of darkness, had to become in a way like other humans. He had to leave behind, so to speak, a lot of His knowledge. In spite of this, He is so great that considerably more knowledge remained with Him than with any other being. Still, the greatest, the last of His knowledge and light, He could not take with Him. Now, these robes are the expression—you might say, the symbols—of His knowledge, His glory and elatedness, which He could not possibly take with Him while He was dealing with the earth sphere and the lower spheres in connection with His task. For then, as you will readily understand now, His task of salvation could not have been accomplished. Only after the task was completed could he return and resume His former state of utter perfection, or reassume what He left behind. And only then would full knowledge would come to Him again.
回答：你们都知道，我的朋友们，精神物品，不管是地形、衣物还是别的东西——人们总是觉得所有这些东西竟然在精神世界或以更精细的材质存在是不可思议的——只不过是当时头脑状态或事情状态的一种表达而已。这种物品不是如你们可能认为的，仅仅向你们人类表达某个意思的象征性物品，而是刚好相反。你们的物品，任何你们在这地球位面上拥有的东西，只不过是精神世界中物品的象征符号而已。换句话说，事情要刚好颠倒过来。现在，从我之前给出的关于堕落和拯救的讲座中，你们应该理解了，耶稣，当祂来到地球位面以及之后前往黑暗位面时，必须以某种方式变得像其他人类一样。祂不得不（可以这么说）放下很多自己的知识。尽管如此，祂是如此伟大以至于祂能留存下来的知识比任何其他存有都要多得多。但是，祂的知识和光明中最重大的、最后的那些，没法被带走。那么，这些长袍就是对祂的知识、祂的荣耀和欢欣鼓舞的表达——你也可以说这些长袍就是那些品质的象征——而当祂处理地球位面和与祂任务相关的低级位面时，这些东西是不可能随身携带的。因为那样的话，如你们现在已经能理解的，祂的拯救任务就无法正常完成了。只有在任务完成之后，他才能回归并重回祂之前的绝对完美状态，或者说重新穿戴起祂遗留下的东西。也只有这时祂的完整知识才会再次回到祂那里。

24.10  The same principle reigns when higher beings of the world of God occasionally visit creatures dwelling in spheres of lesser light and development for the purpose of helping them. When they manifest, they appear to these beings not as the angels of God they are, but have a similar outer appearance to those they visit. They will, with rare exceptions—and these exceptions exist too—not manifest in their full glory, in their light, in their beauty, which also means the beauty of spiritual robes. The moment higher beings descend into darker worlds, their appearance automatically, according to law—unless special measures are being taken for special purposes—alters and assimilates the appearances of the surroundings. Through this process some of the knowledge remains behind—not all, but some. It has to be that way, for the simple reason that if the lower-developed beings would see an obviously higher spirit—an angel for instance—they would without question and without a doubt flock to it and follow it. These creatures are obviously not happy, and seek salvation. And as you humans also constantly hope for, they too desire salvation by outer means. They would say:  “Ah, this is an angel of God. And therefore this is good.”  But actual salvation can only lie in self-recognition and self-purification in whatever world you live in. Therefore, a being has to learn first of all to discriminate independently, to think and choose freely, out of his or her own accord. They cannot do this if there is an influence by a glorious apparition. And this is why, for instance, also on this earth sphere, people of higher and lower development live together with all the grades in-between, having no outer mark that indicates their spiritual standing. People have to learn to make their own choices as to whose influence they are willing to accept by weighing the meanings of the various influences. The choice has to come freely by proper and independent discrimination. As I said, there are cases when an angel of God can manifest in your world, as well as in the beyond, but these are particular circumstances that warrant these measures. Yet as a rule, whenever a higher being manifests to beings of lower development, they show themselves in outer form as those beings for these very good reasons. And it was the same principle with Jesus before he had completed His task. Is that clear?  [Yes.]
相同的原则在上帝世界的高等存有们偶尔以帮助它们为目的而拜访那些居住在缺乏光明和发展的位面中的生物们时同样适用。当它们（高等存有们）显现自己时，它们不会把自己显现为上帝的使者（尽管它们确实是），而是采取一种与它们拜访的生物们相似的外形。它们在罕见特例下——这些特例也是存在的——会以自己全部的荣耀、光明和美——这种美也意味着精神长袍的美——来显现自己。在高等存有们下降到黑暗世界的时刻，他们的外观会自动地，根据法则——除非为了特殊目的采取特殊措施——发生变化并与周围事物的外观相适应。在这个过程中一部分知识会留下来——不是全部，但是会有一部分。事情必须是这样，原因很简单：如果发展较低的存有们看到一个显而易见更高的魂灵——比如说一个天使——它们就会没有任何问题、不抱任何疑问地，成群结队地蜂拥而至，然后追随它。这些生物显然并不开心，而且寻求拯救。而就像你们人类一直期望的那样，它们也渴望通过外部手段得到拯救。它们会说：“啊，这是一个来自上帝的天使。因此这一定是好事。”然而实际的拯救只能通过自我认知和自我净化得到，不管你生活在什么世界中都是如此。因此，一个存有的第一要务是必须学会通过他或她自己的意愿，独立地分辨是非、思考和做出自由选择。如果周围有一个辉煌的特异景象的影响，它们就没法这样做。于是这就是为什么，比如说，在地球也是这样，发展更高和更低的人以及所有中间层级的人们居住在一起，没有外部记号表明他们的精神地位。人们必须学会通过衡量许多种影响力的含义来做出自己的选择，进而决定接受哪一方的影响力。这个选择必须通过适当的、独立的分辨能力而做出。如我所说，有的情况下上帝的天使可以在你的世界中以及“之上”的世界里显示自己，但这些是特殊情况，所以这些措施是正当的。但一般原则是，每当一个高等存有向较低发展程度的存有们显示时，它们都会出于上面解释的合理原因以外部形态展示自己。当耶稣完成祂的任务之前这条相同的原则也是适用的。这样说清楚了吗？【是的。】

24.11  QUESTION:  Are the disciples of Jesus powers which do not have to reincarnate again and do they represent in a symbolic form psychological aspects of human types?
问题：耶稣的信徒们是不是不再需要重复降世了？他们是否以象征的形式代表了各种人类类型的不同心理侧面？

24.12  ANSWER:  There are two questions here. To the first question I say:  There are some among the disciples who do not have to come back anymore and some others who do, but they are very highly developed now and have great tasks to fulfill on this earth. I explicitly beg you, my friends, not to ask me which of the disciples do not have to reincarnate anymore and which of them may be alive now. I have very good reasons, particular reasons, why I do not want to discuss this. And it is not important for your development to know this.
回答：这里有两个问题。对第一个问题我说：信徒中有些人不需要再回来降世了，也有些其他人仍然需要回来，但他们现在的发展程度很高，而且在这个地球上有很重要的任务要完成。我明确地请求你们，我的朋友们，不要问我信徒中哪些人不再需要降世了，哪些人现在可能正在活着。我有非常合理的理由，具体的理由，（来支撑）为什么不想讨论这个问题（的态度）。而且这个问题对你们的发展来说不重要。

24.13  As far as the psychological aspects are concerned, I would like to say this:  The whole Bible, the Old as well as the New Testament, can be interpreted on many levels. The lowest level would be the historical one. There are, of course, many errors and many omissions, historically, which have to be expected. Then there is the level of spirituality and symbolism—the level that you might call metaphysical. And there is—and this is, perhaps, the most useful for human beings in your present state of development—the psychological level. For everything mentioned in the Holy Scriptures also has this level, in addition to the others. One level does not exclude the validity of the other. And even though many of the personalities in the Scriptures were actual persons—not all of them, but many of them—they, at the same time, represent psychological aspects. Because of the existence of these different levels simultaneously, the Holy Scriptures are such a magnificent, outstanding and unique document. Meaning is to be found on each of these planes. It is inconceivably “artful”—to choose this word—to have the Bible constructed in this way. You can never know how strongly and resourcefully God’s Spirit World has actively helped to create this marvel, already foreseeing the many human errors that inevitably must slip in, in the course of time. Despite these errors, the Bible is something that has never been duplicated, but there are extremely few people, indeed, who understand the Bible from this point of view. Many perceive one level, perhaps a few even two levels, but there is hardly a person who can grasp all the levels contained in it.
至于其中涉及到的心理侧面，我想这样说：整个《圣经》，包括《旧约》和《新约》，可以从很多层次上进行解读。最低的层次就是历史层次。当然，这里有许多历史上的错误和遗漏，这是意料之中的。然后就是精神/心灵和象征层面——也就是你们可以称为形而上/超自然的层面。再然后——而这个层面，也许是对你们目前发展状态的人类们来说最有用的——就是心理层面。因为圣典中提到的所有东西在包含了其他层次之外，也都有这个层次。一个层次不排除另一个层次的有效性。同时，尽管圣典中的许多人物都是实际的人——不是全部，但许多都是——他们同时也代表了心理侧面。因为这种不同层次同时存在的特性，使圣典成为一份如此宏伟、卓越又独特的文献。（人）在这些层面的每一层中都可以找到含义。《圣经》的这种构造方式具有不可思议地——选择这样一个词——“艺术性”。你们永远不会知道上帝的精神世界，早已预料到随着时间推移必然会掺杂进的许多人为错误，是怎样强烈地、足智多谋地积极帮助创造了这个奇迹。尽管有诸多（人为）错误，《圣经》是一种从来没能被复制的东西，但是，确实只有极少数的人能够从这个角度理解《圣经》。许多人只觉察到一个层面，也许有些人甚至能觉察到两个层面，但几乎没有人能领悟到它包含的所有层面。

24.14  QUESTION:  The names—I think you answered that now already—of the I.A.O., Sabaoth, Barbelo, etc.; of the “Pistis Sophia”—are they of any importance to discuss?
问题：有些名字——我觉得你已经回答了这个问题了——比如I.A.O.，Sabaoth，Barbelo，等等；以及“Pistis Sophis”——有必要对它们进行讨论吗？

24.15  ANSWER:  No, not presently. Perhaps at a later date.
回答：不，现在不需要。也许之后某个时间可以。

24.16  QUESTION:  In the last lecture you explained that the physical existence on this earth sphere for the fallen spirits was not chosen in an arbitrary form but is, in fact, a result of the degree of density they had by that time. Was there a reverse procedure of this when the Fall occurred?  Or was it sudden?
问题：上次讲座中你解释了堕落魂灵们在这个地球位面上的物质存在形式不是随意选取的，而是由它们当时拥有的密度程度决定的。当堕落发生时，有没有一个这个过程的逆过程呢？还是说它是突然发生的？

24.17  ANSWER:  No, it was not sudden. It cannot be sudden. I even mentioned that the Fall was also a very gradual process. But the earth sphere did not come into existence in the process of the Fall. Its existence is a result of redevelopment. The various stages of the Fall manifested in other forms. In explaining part of this, I will take care of another question on the agenda. Some of you have wondered whether a spirit, before being ready for incarnation, has to go through these various stages.
回答：不，不是突然发生的。它不会是突然发生的。我甚至提到过堕落也是一个非常渐进的过程。但地球位面不是在堕落的过程中形成的。它的形成是“再发展”过程的结果。堕落有许多阶段，这些阶段以其它形态显现。在解释这个问题的一部分的过程中，我会同时处理另一个本来要后面说的问题。你们有些人已经思考过“在一个魂灵准备好进入降世过程之前，是否需要经历这些个阶段”这个问题了。

24.18  Here are the facts, as well as I can transmit them to you, at least in a condensed form:  I have mentioned that in the Fall, a plurality came into existence from a unity. In other words, a splitting occurred. It is not only that the one being, the dual being, split into half, but as the Fall continued, the split multiplied and multiplied. And some of these part-souls became, as an expression, mineral, plant and animal life. Before the earth-sphere existed, these expressions existed in other worlds or other forms, and when the world of matter came into existence, these part-souls incarnated in these various forms.
事实是这样的，我会尽可能地、至少是以一种压缩的形式把它们传递给你：我已经提到，在堕落过程中，从统一体中产生了多元化/复数体。换句话说，产生了一种分裂。这种分裂不仅是说一个存有，一个二元存在，裂成了两半，而是说随着堕落的继续，这种分裂成倍成倍地增加了。这些“片段灵魂”中，有些——作为表现形式——成为了矿物质、植物和动物形态的生命。在地球位面形成之前，这些表达形式在其他世界里或以其他形态存在过，而当物质世界形成时，这些“片段灵魂”以这几种形态进行了降世。

24.19  In other words, mineral, plant and animal life on earth are, to some degree at least, expressions of manifestation on the downward curve until they reach the stage where they are ready for human incarnation, with longer or shorter periods in-between—this varies with the individuals—of existence in lower worlds than the earth sphere. The higher the curve goes upward, the more these particle-souls unite again. For instance, the split is stronger in the mineral kingdom than in the plant kingdom, and the latter is a further split than the animal kingdom. The part-soul does not always have to go through the same forms of existence on the downward curve as on the upward curve, but sometimes it must go through the same types of incarnation. This varies and happens perfectly and minutely according to law.
换句话说，地球上的矿物、植物和动物形态的生命，在抵达一个能够做好准备进行人类降世，并在中间夹杂着在比地球位面更低的世界中进行或长或短的存在——这个情况根据个体差异而不同——过程的状态之前，至少在某种程度上，都是下行曲线上物质显现的表达形式。曲线越向上，这些“片段灵魂”就会发生越多的统一。比如，矿物王国中的分裂比植物王国更深，而后者又比动物王国的分裂更深。处在下行曲线上的“片段灵魂”与处在上行曲线上的部分—灵魂相比，不总是需要经历相同的存在形式。但有的时候它必须经历相同类型的降世过程。这个情况不一而同，而且都根据法则完美地、精确无误地发生。

24.20  Now you may wonder about this insofar as you have beautiful mineral, plant, and animal life, and why these forms of existence should in many cases be still on the downward curve and thus be further back in development than some souls who are obviously in a less harmonious state. To this, my answer is:  Not all beings are equally guilty in the Fall. And besides, it was never the entire personality of the unified beings that was responsible for the Fall, but certain personality trends which lend themselves to the deviation from divine law. Nevertheless, the entire being fell.
现在你们可能觉得奇怪的是，你们有丰富而美丽的矿物、植物和动物生命形态，那为什么这些存在形态在很多情况下仍然处于下行曲线上，因此相比某些显然处于更不和谐的状态中的灵魂们处于更落后的发展阶段。对此，我的回答是：不是所有存有在堕落中都有相同程度的罪过。而且，从来都不是一个存有的整个人格个性要承担堕落的过错，而是某些特定的个性趋势使它们自己产生了对神圣法则的偏离。但尽管如此，整个存有都堕落了。

24.21  Now, the beautiful and harmonious manifestations in nature—whether on the downward or on the upward curve—are expressions of the parts of the spirit that were not, or only to a lesser degree, involved in the deviation from divine law. You all know that certain animals, for instance, have been and still are dying out and other forms of animal life come into existence. The same applies to plant and mineral life. This is due to the fact that the expression of manifestation is also influenced by other beings who, as you know, are able to create worlds and types of manifest life-force. The more your desires and goals are purified, the more you automatically contribute at the same time to the creation, and thus are able to change it for the better, even as far as the part-souls on the down-curve are concerned. Can you grasp at all what I am saying?  [Yes.]
于是，自然中存在的那些美丽而和谐的物质显现——不管处在下行还是上行曲线上——都是这个魂灵中没有，或者说只在较小程度上，参与了对神圣法则的偏离的部分。比如，你们都知道某些动物曾经，以及现在都发生过灭绝，而其他形态的动物生命不断形成。这对植物和矿物生命同样适用。造成这种现象的原因是物质显现的表达形式也受到其它那些有能力创造不同世界、不同种类的生命力实现形式的存有们的影响。你的欲望和目标越纯净，你就能越加自动地同时为宇宙造物做出贡献，以此把它变得更好，哪怕对那些仍然处于下行曲线上的部分—灵魂来说也是这样。你们能领会我说的（哪怕）任何（一点）东西吗？【是的。】

24.22  QUESTION:  It is terribly difficult to understand, but I somehow grasp that they have to collect themselves later in order to incarnate as humans?
问题：这部分内容极为难懂，但我似乎领会到那些“片段灵魂”后来必须收集/整合自己才能以人类形态进行降世？

24.23  ANSWER:  Yes. The fluids of the several particles will connect when they reach a higher form of existence, just as when the dual beings, once having reached that state of perfection, will flow together and become one. It is the same process in principle.
回答：是的。当这些颗粒抵达一种更高形态的存在时，它们的流体会进行连接，这就像当二元存有抵达那种完美状态时，会流到一起然后合二为一。原则上这是相同的过程。

24.24  QUESTION:  Yes. And my question was:  The spirits in darkness, do they have to go through all these stages and then…
问题：是的。我的问题是：那些黑暗中的魂灵们，它们是否需要经历所有这些阶段然后…

24.25  ANSWER:  They do not have to go through all these stages again, but just some of them. This is again according to individual characteristics and individual considerations. I could not possibly go into all the details of what is considered, how this works, etc. You could never understand it.
回答：它们不需要再次经历所有这些阶段，而是只需要经历其中一些阶段。这也是要根据个体特征和具体情况考虑的。我不可能详细展开解释所有考虑到的细节是什么，其中的运作原理是什么，等等。你永远没法理解那些。

24.26  QUESTION:  Between incarnations, is the density there indicative of the development?  Can one become suddenly so high or so low that one cannot see them?
问题：在两次降世之间，那（种环境）里的密度等级是否能说明（这个人）相应的进化阶段？一个人能否突然变得很高或很低以至于让人看不到？

24.27  ANSWER:  In ordinary circumstances, unless special measures are taken, as I said, you can only see those of your kind—in spirit too, unless you reach a certain stage of development. Then you can see those which are of lesser development. [Does that mean that there is an equivalent of spirit density for the incarnated density?]  Certainly. As I explained last time, your physical matter with this kind of density is one type. And spiritual matter, according to the various spheres, is of another type. It is all matter in different degrees of density. It is only for your human conception that you think this is physical and then you imagine just one other category—the spiritual. But this is wrong. There are all different degrees. Does that answer your question?  [Yes. Thank you.]
回答：在通常情况下，除非采取特殊措施，如我所说，你只能看到与自己同一类型的（魂灵，以及密度等级的物质）——在精神世界也是这样，除非你抵达了一个特定的发展阶段。这时你就能看到所有较低发展阶段的密度等级。【这是不是说有一种与降世密度等级相符合的精神密度等级？】当然。如我上次解释的，你在这种密度下的实体物质是一个种类。而精神物质，与许多（你所连接）不同的位面相对应，是另一个类型。一切全都是不同密度等级的物质。只有在人类概念中你们才认为（你们眼前的东西）这是物质的、实体的，然后你们构想出单单一种类别的物质——精神物质。但这是错的。实际上有所有不同的程度等级。这样说回答了你的问题吗？【是的。谢谢你。】

24.28  QUESTION:  How can a dormant power of concentration be revived?
问题：怎样重新激活一个休眠中的集中注意力方面的力量？

24.29  ANSWER:  This is very individual, but in your case I would like to give you special advice and I will need a little time—just a moment. This is what I have to say:  There is both a great strength and a little tense strength in you. As long as you do not relieve the tension, it will be difficult for you to awaken the strength. In order to do that, you will need to take certain measures yourself. And when you do that, you will actually begin to feel this tenseness and when you do feel it, then you will be a step nearer to releasing this clot you have in your soul.
回答：这是非常个体化的，但在你的情况下我想给你特殊的建议，我需要一点时间——只是一小会。我要说的是这样：你内心有一种很强的力量，同时也有一点紧绷的力量。只要你不释放这种紧绷的张力，想要唤醒这份力量就会很困难。想要做到这一点，你需要自己采取一定的措施。当你这样做时，你将能够感觉到这种张力，当你感觉到它时，你就离解除自己灵魂中的这个凝块更进一步了。

24.30  As long as you cannot clearly feel it, you will be unable to eliminate it, for you cannot handle something you are not aware of. You may be aware now of the existence of it by its symptoms, but you do not understand yet what it consists of. And this is why you have to find out. Your past endeavors of concentration were going in a wrong direction, my friend. You will have to change the direction of your search, that is, you have to probe into your own soul, getting to know yourself fully. Forget about your inability to concentrate generally for the time being and start searching for your own soul, your own innermost feelings and motives. This is the only way you can dissolve the knot in you. This is the only way you will begin to feel the almost physically appearing clot in you. And when you find it, do not push it back, thinking you can eliminate it that way. On the contrary, welcome it like a long-lost friend and draw it out into consciousness.
只要你还没法清楚地感觉到它，你就没法消除它，因为你没法处理自己觉察不到的东西。你现在也许能根据它的症状觉察到它的存在，但你还不理解它是由什么组成的。而这就是你需要发现的东西。你之前进行集中注意力的努力用在了错误的方向上，我的朋友。你需要改变搜索的方向，也就是说，你需要在自己灵魂中摸索，完整地了解自己。先忘掉自己目前无法集中注意力的这种大致情况，然后开始寻找自己的灵魂，寻找你自己内心最深处的感觉和动机。这是消除你内心绳结的唯一方式。这是你开始感觉到自己内部几乎实际出现的凝块的唯一方式。当你找到它时，不要把它推回，以为你可能以这种方式消除它。相反，像一个失散多年的老朋友一样欢迎它，并把它带到清醒意识中。

24.31  You may ask:  How do I go about it?  This I could not possibly show you even in an entire lecture, but you will find the answer, my friend, by following my instructions in the series of lectures I will begin shortly. I mentioned earlier that I will start this course through which all my friends who do not attend private sessions will be able to proceed step by step in the proper direction of inner self-development, of making hidden emotions, fears or problems conscious and dealing with them properly. Your question now touches merely a symptom of an entire psychological block and this cannot possibly be solved by one answer or simple advice I could give you now. This you must understand.
你可能会问：我具体怎样进行呢？这个问题就算给我一次完整讲座的时间也不够用，但你会找到答案的，我的朋友，只要遵循我不久就会开启的下一系列讲座中的指示就可以了。我之前提到过我会开启这份课程，这样所有不参加私人集会的朋友们就都能一步一步地行进在正确的方向上，有条理地处理内心的自我发展的问题，并把内心的隐藏情绪、恐惧心理和相应的困难带入清醒意识，并合理地处理它们。你现在的提出的这个问题触碰到的只不过是一整块心理障碍所表现出的症状而已，而这是不可能通过我现在给出的单独一个回答或简单建议就能得到解决的。你肯定理解这一点。

24.32  The only thing I can tell you now is that you have searched in the wrong direction. You have to search much nearer, in yourself. You have to start on the road of self-recognition in a much more profound way. It is a new way of life and you will be able to get really started when we begin the course. It will mean a lot even to those friends who come privately to work with me. This will not be a waste of time for them, but will be an additional help, even though some of what I will say will not be quite new to them. Then you will learn, my friend, how exactly to go about dissolving this clot or block, step by step. You cannot do it by one single measure, by a certain prescribed formula, but it is a new way of life, as you will see, too. Your spiritual foundation is a good one and this helps. But it is something psychological that is in your way. I mean to concentrate on this in the course we will begin. As long as the psychological blocks are not dissolved, the spiritual development cannot go on—or if it does go on in some way, it is not a thorough or an efficient way. There is then always a discrepancy that you have to iron out sooner or later. If you do not, it will finally backfire. There has to be harmony between the spiritual and the emotional levels. For the time being, it is important for you to realize that you cannot dissolve such a psychological block in any other way than the one I will guide you on in this group; that you have so far searched in the wrong direction and have to alter the direction by asking yourself about yourself and giving yourself extremely honest answers. Meditate on this. Find out what your fears are, what you really desire, who you really are.
我现在唯一能告诉你的是，你之前是在错误的方向上搜索。你必须在离自己更近的地方、在你自己内心搜索。你必须以一种更加深远的方式开启自我认知的道路。这是一种新的生活方式，当我们开始随后的课程时你就有能力真正开始这样做了。这（系列课程）对那些到我这里开展个人工作的朋友们来说也会有很大意义。尽管有些我要说的话对他们来说不会特别新鲜，但这对他们来说不会是一种对时间的浪费，而会是一种额外帮助。这时，我的朋友，你就能具体地、一步一步地学会怎样化解这个凝块或者说障碍。你没法通过某一个单独的措施、某种开具的配方达到这个效果，因为这是一种全新的生活方式，你也会看到这一点。你的精神基础是好的，这一点会有帮助。但是阻碍你的是某种心理层面的东西。我打算在即将开始的课程中着重关注这一点。只要心理层面的障碍物还没有化解，精神发展就没法继续——或者说即使它在某种方式上可以进行，那也是不全面或不高效的。因为这样的话就总会产生某种不协同/不一致的地方，而你早晚都必须排除掉它。如果不排除，它就会最终适得其反。精神层面与情绪层面之间必须存在和谐。目前来说，重要的是你要意识到，要化解这样一个心理障碍，你只能通过我将在这个小组里指引你的这种方式排除它；目前为止你一直在错误的方向上搜索，因此必须通过询问自己内心并给自己提出极其诚实的回答的方式改变之前的错误方向。对（我现在说的）这个问题进行冥想。找出你自己的恐惧，找出你真正的渴望，找到真正的自己。

24.33  That will get you started on this road. If you succeed in finding out something about yourself in the meantime, if you can answer some questions that you have not answered so far and thus begin to gain greater self-knowledge, you may occasionally come to feel this block very clearly and what its real significance is. This block does not merely consist of one thing, but of a number of interacting subconscious tendencies, fears and resistances. These use up your inner strength and, among other things, prevent you from being able to concentrate as much as you desire.
以上内容会让你开始走在这条路上。如果你在这段时间里成功地找出自己的某些问题，如果你能回答某些自己目前为止没法回答的问题，并以此获得更深的自我认知，那么你就能偶尔非常清晰地感觉到这份障碍，以及它真正的重要性是什么。这份障碍不是单单由一件事组成，而是由一系列互相作用的潜意识中的趋势、恐惧感和阻力组成的。这些东西占用了你的内部力量，同时，包含但不限于此的是，阻止你如愿达成相应的集中注意力的程度。

24.34  You are not satisfied with my answer, I know, my friend. It is not what you would have liked to hear. You would have liked to receive one ready-made recipe, supposed to dissolve your difficulty with one sweep. But, unfortunately, it can never be done that way. It is only through the long road of self-knowledge and self-recognition that the answers will gradually make a whole, the answers that you yourself have to find. Everything else is not truth and therefore I cannot say it. But where humans do need help is to know how to go about it, and that you will learn gradually by following this course. There is a great possibility in you, spiritually speaking, but this is blocked by psychological subconscious trends which can be dissolved only in the above-mentioned way. There is no ready-made formula. That is the truth, my friend, and I know you do not like to hear this. I can do you much more good telling you the truth than telling you something according to your expectancy. And you have the means at your disposal to find out that it is so.
你对我的回答不满意，我知道，我的朋友。这不是你想听到的东西。你想听到一个准备好的配方，它能够把你的困难一扫而光。然而不幸的是，问题不可能这样解决。只有通过漫长的自我知识和自我认知的道路，你需要的答案才会慢慢地组成一个整体，而这些答案是需要你自己找到的。任何其他东西都不是事实，因此我没法那样说。但人们在这个过程中确实有需要帮助的地方，那就是其中的具体细节如何展开，而这一点你通过跟随这份课程是可以逐渐学会的。从精神角度来说，你内心有一份很大的可能性，但这份可能性被一些潜意识的心理层面趋势阻碍住了，这些趋势只有通过上面提到的方法才能化解掉。这里没有现成的配方。这是事实，我的朋友，而我知道你不愿意听到这些。相比根据你的期望告诉你一些想听的东西，告诉你事实会好很多。而你目前有能力发现事实确实是这样。

24.35  QUESTION:  First, I would like to ask if you can see my thoughts.
问题：首先，我想问下你能否看到我的思想。

24.36  ANSWER:  Not while I am in the medium. For, while I manifest through a human being, the same matter stands in my way as the matter you have in your way while you are in the body. But when I am out of the body, then I can see thoughts. But mostly I will not give an indication of this. There are times, again according to law, when I may give hints—and only the person himself or herself will know about it. But mostly I will not do so, because, you see, we have to be discreet about this. We cannot violate your free will. If you choose to mention something, then it is a different story. But as long as you choose to keep something secret, I have no right to violate your free will.
回答：当我处在这个媒介中时我看不到。因为，当我通过一个人类个体显现自己时，相同的物质阻挡着我，这就像你在自己身体里时受到物质阻挡一样。但当我处在身体之外时，我就可以看见（人们的）思想。不过绝大多数情况下我不会表明这一点。有的时候，同样是根据法则，我可以给出提示——只有当事人他或她自己会知道的相应的提示。但多数时候我不会这样做，因为，你看，我们必须对此保持慎重。我们不能侵犯你们的自由意志。如果你自己选择提到某些东西，这就是另外一回事了。但只要你选择对某件事保密，我就无权侵犯你的自由意志。

24.37  QUESTION:  I was wondering about what goes on outside of the Plan of Salvation. Are there any new spirits being created and do pure spirits still fall or do purified spirits still fall anew?
问题：我想知道在拯救计划的框架外有没有什么事情发生。有没有新的魂灵被创造出来，以及纯净的魂灵会不会仍然进行堕落，或者说已经净化完成的魂灵再一次堕落？

24.38  ANSWER:  A re-purified spirit cannot fall. That is impossible. And whether the spirits who have not fallen so far or those who will be created later will fall in the future, that, my friends, I do not know. And no one knows that. But if they do, the Plan of Salvation is there and they will not have to go through certain stages again that have been accomplished already once and for all. New spirits are not being created for the time being, not until this Plan of Salvation has found its conclusion.
回答：一个重新净化后的魂灵无法（再次）堕落。那是不可能的。至于目前为止还没有堕落太远的魂灵或者那些以后会被创造出来的魂灵未来会不会继续堕落，这一点，我的朋友们，我不知道。没有人知道。不过如果它们堕落的话，由于拯救计划存在，它们不必再次经历某些阶段，因为这些阶段已经被永久性地彻底完成了。目前来说，没有新的存有正在被创造出来，在拯救计划完结之前都不会有新的存有被创造出来。

24.39  QUESTION:  What is the majority at present?  The pure spirits or the fallen spirits?
问题：目前来说谁是大多数？是纯净的魂灵还是堕落的魂灵？

24.40  ANSWER:  This is really unimportant for you to know.
回答：这对你们来说真的不重要。

24.41  QUESTION:  The Book of Exodus says that the people were told to collect manna only for one day and on the Sabbath for two days. If they collected for two days on any other day but for the Sabbath, it rotted but for the Sabbath it did not. What is the meaning of this?
问题：《出埃及记》中提到人们被要求只花一天的时间采集吗哪，而在安息日的时候要花两天收集。如果人们在任何不是安息日的日子中花两天时间采集吗哪，那么这些吗哪就会腐烂，但在安息日这一天不会有这种情况。这其中的意义是什么？

24.42  ANSWER:  The manna is the symbol for spiritual strength, spiritual truth, divine blessing, all the material you need to advance spiritually, to find yourself and God. With the people of the best intentions—the most diligent workers in God’s vineyard—it is often timing that is so important:  the proper distribution, for instance, of the active and passive forces. I will soon give a lecture on this subject. Both forces have to be utilized harmoniously in the human soul so that each fulfills its function properly. Often one side of your nature tends to be overactive in the wrong way while your other side is overly passive, again in the wrong way. When you are spiritually active, you tend to hoard the strength you may need for the morrow, or the knowledge that you may need for the morrow. This cannot be done. The text you quoted says, in different words, that you have to live in the moment, or what you call the Eternal Now. Each moment has its own requirements, and meeting them can only be done by living completely in this moment. It also says:  you should not take on more than you can chew at the present time.
回答：吗哪象征了精神力量、精神真理、神圣祝福，以及其他一切你为了取得精神上的进步、去找到自己和上帝，所需要所需物质。对拥有最虔诚意图的人们——也就是上帝的葡萄园中最辛勤的工人们——来说，通常最重要的东西是时机：比如说，对主动和被动力量的合理分配。我很快会就这个主题给出一个讲座。两种力量都必须在人类灵魂中被和谐地利用起来，这样每种力量都能合理地发挥它的功能。经常发生的情况是，你本性的一面会在错误的方向上变得过度积极，而在另一个方面（同样以错误的方式变得）过度地消极。当你在精神方面很积极时，你倾向于对明天可能需要的力量进行囤积，或者是囤积明天可能需要的知识。这是不对的。你引用的文段，用不同的话语表达的意思是“你必须活在当前的瞬间”，或者是你们称为永恒的现在的地方。每个瞬间都有它独特的需求，而要满足这些需求，人必须完全地生活在这个瞬间当中。它同时也说：你不应该在当前时间承担起超出自己消化能力的东西。

24.43  However, you do need a little reserve at certain times for the periods when you cannot muster the strength to have inner or outer activity. The Sabbath, as you know, signifies among other things the day of inactivity, the day of rest. In life, everyone has to go through periods when they cannot muster the force to be active. They are tired, they have to rest. And this may also be good spiritually. Everything absorbed in the period of activity has to be assimilated in the period of passivity. And for these times you need a little reserve, but ordinarily, if you feel in the full strength of active life, spiritually, physically, emotionally, on all levels, you cannot possibly hoard. Human beings do that often, again on all levels. They are so anxious, they are so full of fear that they do not trust God, do not trust the harmony of their own innermost selves that will fit into the scheme of divine law, that will go with the stream. They think they have to take care of the future. By this I do not mean that you should be careless. No extreme is ever right. But live in the now and make the best of each moment. Then your manna will be always fresh and will be given to you every day anew. And simply because you live that way, when the next passive period comes around you will quietly nurse what has so beautifully grown during the active period. You will instinctively sense that you have received enough. But this will happen only when you live in harmony with the movement of the active and passive streams of your personal life, only when you have refined your inner senses so much that you feel clearly what each period signifies:  the active or the passive—the weekday or the Sabbath. The analogy applies also to the duration of the periods:  the active periods have to be longer than the passive ones, although the latter always have to reoccur regularly.
不过，在有些时间里你确实会需要一点储备，来应对那些你无法鼓起勇气进入内部或外部活跃状态的时期。安息日，如你们所知，代表了（包括但不限于）静止的、不活跃的那一天，也就是休息的那一天。在生活中，每人都必须经历一些时期，这时他们没有足够的力量进入活跃状态。他们疲劳了，因此需要休息。从精神层面上来说这也可能是好事。在活跃时期中吸收的一切东西，都需要在静止被动的时期中得到消化和整合。因此对于这些时期，你会需要一点储备，不过通常来说，如果你从精神上、身体上和情感上，在所有层次上都感受到了活跃/主动性生命力的全部力量，你就不可能进行囤积。人们经常地，在各种层次上这样做。他们如此紧张不安，他们充满恐惧，以至于他们不信任上帝，不相信自己内心最深处的和谐会划入神圣法则的设计中、会溶入某种更大的趋势中。他们认为自己必须安排好未来。我这样说的意思不是说你们应该（对未来）漫不经心。走极端从来都是不对的。而是说要生活在当下，并且把每个瞬间做到最好。这样你的吗哪就总会是新鲜的，并且每天都会重新给你。这样，因为你是以这种方式生活的，当下一个被动周期到来时，你会静静地培育在活跃时期中如此美妙地成长起来的东西。你会本能地感觉到，自己已经接收到了足够多的东西。但这种情况只有当你以和谐的方式度过自我生命中主动/活跃和被动周期的交替时、只有当你把自己的内部知觉精炼到这样的程度，以至于你能清晰的感觉到每个时期代表的内容：活跃、主动的和静止、被动的——工作日和安息日时，才会发生。这个类比也适用于这些时期的持续时间：活跃周期必须长于被动周期，不过后者总是会有规律地反复出现。

24.44  QUESTION:  The law is, you have to give up what you want to gain. How can we be certain that we are giving up so completely that the idea of gain is not somewhere hidden as a subconscious motive?
问题：法则说，你必须放弃自己想要的东西，这样才能真正得到它。我们怎样确定自己放弃得非常彻底，并保证去进行获取的想法不会隐藏在暗处，成为一种潜意识动机？

24.45  ANSWER:  The fact that you can ask and consider such a question is already the first step to assure yourself that you are on the good road. You have to be ever wakeful for these hidden motives. Test yourself, check yourself each day in your hour of prayer, meditation and daily review, when something like this comes up. “Oh, here my ego intrudes again. I have to give up craving recognition by others—or whatever it may be. Then ask yourself this very question:  “Am I really willing to give it up, or do I do it because I hope to gain something?”  In other words, see if the truth is that you are not able to give it up at all.
回答：你能提问并考虑这样一个问题，这本身已经是“确保自己处于正确道路上”的第一步了。你必须时刻保持对这些隐藏动机的警觉性。当这类事情出现的时候，就在自己每天祈祷、冥想和进行每日回顾的时候测试自己、检查自己。“哦，我的‘小我’在这里又插手进来了。我必须放弃被别人认可的渴望——不管这种认可是什么都一样。”然后这样问自己：“我真的愿意放弃它吗，还是说我这样做是因为我希望得到某种东西？”换句话说，看看事实是不是你还无法真正彻底放弃它。

24.46  If you really want to know the truth about yourself, you can always find it out. This is merely a matter of whether you really desire the truth about yourself or not. If you are unwilling to face yourself with the truth, and rather believe you are living and feeling and thinking in this or that respect according to law, then you will not find the truth. But if you realize that, it is much better. It is the only way, indeed, to recognize your present inability to give something up. It is the only way, for instance, to accept your imperfection in this respect, in humility, yet with the sincere desire to be able to change it, realizing that you need your own goodwill to do so—your own constant self-honesty—and how far you are still deviating from the right course. Realize at the same time that you cannot do so without the help of God, for which you should ask specifically every time you recognize your inability in this or any other respect.
如果你真想知道关于自己的事实，你总是可以找到它。这仅仅是一个“你是否真正渴望关于自己的事实”的问题。如果你不愿意使自己面对这个事实，而是（一厢情愿地）相信自己在这个或那个方面（已经）是以符合（精神）法则的方式生活、感受和思考的，那你就不会找到这个事实。但如果你意识到了这一点，事情就会好很多。事实上，这确实是意识到“自己目前还无法放弃某件东西”这个情况的唯一方法。这是唯一的，比如说，接受“自己在这方面仍然不完美”这一事实的方法：（也就是说）既抱有谦逊态度，同时，还真诚地渴望有能力去改变它，并且意识到，你需要自己的良好意愿——你自己持续的自我诚实——来做到这一点，以及你仍然与正确轨道存在多少偏离。同时也要意识到，你无法在没有上帝的帮助的情况下做到这一点，而每当你意识到自己在这个或那个方面能力不够时，你都应该向上帝提出具体、详细的请求。

24.47  The knowledge that you are still not giving up what you want to gain, even though a part of you desires it, is the best medicine. If you can recognize that again and again, and then ask God for enlightenment, strength and higher understanding, to help you want with every particle of your being to purify your motives, then you will eventually succeed, little by little. It will happen not in one sweep, but first just in a small measure, occasionally, and as time goes on and you do not lessen your efforts—which should never be tense—you will find it easier and finally it will be your natural reaction.
有了这种“你仍然不愿意放弃你想要得到的东西”的知识，那么哪怕你内心中的一部分仍然渴望它，你已经有了最好的解药。如果你能一次次地认识到这一点，然后向上帝请求启迪、力量和更高的理解力，来帮助你以全身心的力量去渴望净化自己的动机，那么一点点地，你最终定会成功。它不会以一扫而光的方式发生，而是一开始，偶尔地，会以很小的程度发生，然后随着时间推移，在你不放松自己努力——这种努力绝不应该是紧绷的——的情况下，你会发现这对你来说变简单了，最后它会成为你的自然反应。

24.48  In the course of this path, you will be guided to situations where you will have an opportunity to do this, when you can prove it by an action. It does not have to be something big and important. Often, the smaller the act, the better it is for you. The less people are aware of it, the better it serves the purpose. No one ought to know but yourself and your spiritual teacher. And if you can really give up without letting the other person involved know, then, once you have tried it, it will be so much easier the next time. This is the way you gradually find the law. Nothing like this can ever be accomplished suddenly or with one single act. It can only grow in constant endeavor, in constant effort. Nothing can be accomplished any other way, nothing that is durable and really solid, least of all spiritual security, harmony and a firm foothold.
在这条道路的过程中，当你能够通过行动证明这一点时，你就会被指引到一些情境里，使你有机会真正这样做。这不一定是某种庞大又显赫的事情。通常，这种行为越小，其实对你越有利。知道的人越少，越能满足这种目的。除了你和你的精神导师之外没人需要知道。而如果你真正能在不让事情中另一个人知道的情况下做到放弃，那么，一旦你做到了，对你来说下次就会容易得多。这就是你逐渐发现法则的过程。这样的事从来没法一瞬间、通过一个单独行为被完成。它只能通过持续的尝试和持续的努力才会生长。没有什么持久的、真正坚固的东西能够以任何其他方式得到，更不要说精神上的安全、和谐和稳固的落脚点了。

24.49  QUESTION:  How shall we feel or react toward injuries—as for instance slander, gossip, malicious talk—with a truly spiritual attitude?
问题：我们应该对（内心）受伤持什么样的态度和反应呢，比如诬陷、说闲话和恶意中伤，怎样以一种真正精神层面的态度对待这些事呢？

24.50  ANSWER:  You all know, the answer is to forgive. But to do it, to be really able to do it, is a different matter altogether. The danger always exists that, in your good intention, you talk yourself into having forgiven, while in reality you are doing no such thing. And that, of course, is much more harmful than knowing you still cannot forgive, knowing this imperfection. So the first step is again self-honesty. Compare your actual feelings to the true course you know to be right. In all matters, this remains always the basis of purification, the first step without which there cannot be a further step.
回答：你们都知道，答案是去原谅。但要这样做，要真正有能力这样做，就是一个完全不同的问题了。总会有一种危险，那就是出于你的好意，你说服自己已经原谅了别人，而实际上你做不到。而这样做的危害，当然了，要比知道自己仍然无法原谅、知道这份不完美所造成的危害大得多。所以第一步还是自我诚实。把自己的实际感觉跟你认为正确的解决方法进行比较。不论什么事情，这种（自我诚实的）做法都是进行净化的基础，如果没有这个第一步，就没法进行下一步。

24.51  Then ask God to help you again to have sufficient vision to be able to forgive wholeheartedly. And try to pray for the person who offended you. This may cause you inner resistance at first, but if you try again, you will finally be able to do so without inner resentment. Furthermore, ask yourself:  “Do I really want to forgive?”  You will then see that part of you wants to do it, but another part enjoys not forgiving. If you recognize that, you have a further clue about the truth of your lower nature—and this is priceless knowledge on the path of purification. Go with this resistance to the Father and ask Him to help you. Realize that you cannot accomplish the least thing without His help. Your part forever has to be goodwill, diligence, and utter self-honesty.
然后再次请求上帝帮你获得足够的视野和远见，从而能进行全心全意的原谅。尝试为那个冒犯了你的人祈祷。一开始这会使你产生内部阻力，但如果你不断尝试，你最终会在没有内心怨恨的情况下做到这一点。进一步地，询问你自己：“我真的想要原谅吗？”你会发现，自己有一部分是想要原谅的，但另一部分更享受不原谅。如果你能意识到这一点，你就对自己低级本性的真相有了进一步的线索——而这份知识在净化的道路上是无价的/无比珍贵的。带上这份阻力去找天父，并请求祂帮助你。意识到如果没有祂的帮助，你连最微小的事情都没法完成。你要做的永远都必须是保持良好意愿、勤奋态度以及彻底的自我诚实。

24.52  Another thing that is equally important, not only in order to forgive, but for your development altogether, is the realization that nothing can ever happen that is not based on the law of cause and effect. By this I do not mean that the other person involved is justified or right. His or her wrong may be even greater than your own. And your own wrong may not be in apparent connection with the incident that is its result at this present time. The connection may be indirect and hard to find. But you need to realize that there must be something in you that has, at one time or another, caused this—some wrong current or attitude, some imperfection in your soul that is responsible for the event. Even if the connection is indirect, it is still according to the law of cause and effect. And if you really and truly wish to find out—without shirking from the truth—and ask God with all your heart to help you, you will receive the answer. Know that there must be something for you to learn and you will learn it.
另一件同样重要的事，不仅是为了去原谅，也是为了你完整的发展，是要意识到从来没有什么事情不是建立在因与果的法则之上的。我这样说的意思不是说涉及到的另一个人是正当的或正确的。他或她的错误可能比你自己的还大。而你自己的错误可能与这个错误在当前时间下导致的结果之间没有明显联系。实际的联系可能是间接的，并且很难找到。但你需要意识到，你内心中一定有某种东西，在某个时间，引发了这件事——某种错误涌流或错误态度，某种灵魂中的不完美之处导致了这个事件的发生。哪怕这个联系是非直接的，它仍然符合因果法则。如果你真心诚意地想要找出它——而不会躲避事实——并且全心全意地请求上帝帮助你，你就会收到回答。意识到肯定有什么东西是需要你学习的，这样，你就会学到它。

24.53  I cannot emphasize strongly enough that the desire has to be wholehearted, not just asking God once perfunctorily. You must be filled with the desire to find out the cause in you. If you overcome the resistance against finding out, if you keep awake through the days for an answer, without tension, just being aware, the answer may come through other people or through something you read or something apparently unconnected. All of a sudden you will feel an inner echo and you will know:  this is God’s answer. Unpleasant as it may be, it will be a great liberation, a great step forward—and this in itself will make it so much easier for you to forgive truly. Then you will be able to embrace the incident, knowing that it helped you upward, forward, nearer to God, to perfection, to happiness that can never be robbed from you. Without this incident you could not have discovered your imperfection and thus you would have remained in a much lower state of awareness, development, and therefore also personal happiness. All this does not only apply to incidents of the kind you have quoted, but to all mishaps. When you come to the point where you embrace the heretofore unpleasant incident because you have understood its healing nature for your mind, body, soul and spirit, then you have accomplished a great deal.
我怎么强调都不过分的是，这种渴望必须是全心全意的，而不是仅仅敷衍地请求上帝一次。你必须充满这种“找出自己内部原因”的渴望。如果你能克服不去寻找它的阻力，如果你在接下来的时间中保持清醒等待回答，没有紧绷的情绪，只是保持觉察的状态，那么这份回答就可以通过某个其他人，或者某个你读到的东西，或某个显然不相关的事情找到你。突然之间你会感到内心中有一种回音，然后你会自然地知道：这就是上帝的回答。这个回答也许会令人不愉快，但它会带来很大的解放，它是向前的一大步——而这本身就会使你去真正原谅别人变得容易很多。这时你就可以完全接受这个事件，因为你知道它帮助你向上、向前，使你更接近上帝、更接近完美、更接近永远无法被剥夺的快乐。没有这个事件，你就不可能发现自己的不完美之处，因此你会保留在一个低得多的意识状态和发展状态中，因此你的个人幸福程度也会低很多。所有这些（情况）不仅适用于你引用的这种事件，也适用于所有类型的不幸事故。当你到达了一个节点，并且能完全接受当前令人不愉快的事件，认识到这类事情对你的头脑，身体，灵魂和精神的治愈作用时，你就已经完成了一件很了不起的事。

24.54  QUESTION:  Between what relationships is there a karmic situation, as, for instance, parents, children, husbands, wives, sisters and brothers?  Does it go further than that?
问题：在哪些人际关系中会有一种因果业报的情况呢，比如说父母，儿女，丈夫，妻子，姐妹和兄弟？还是说这种情况会继续延伸得更远？

24.55  ANSWER:  Oh, you cannot possibly make a rule on that. In the majority of cases, there are karmic bonds in the immediate family, but not always. It may very well be that there is no karmic relationship with some members of the family. It may be a new relationship that was given because it best fulfills the purpose of the lives of all concerned. And there may be a number of karmic relationships that are not within one’s own family. They may exist with people one meets in the course of one’s life, sometimes even quite late. For all that there is a good reason and purpose.
回答：哦，你不可能在这种情况上做出一般性概括。在大多数情况下，直系亲属之间会有因果业报的纽带，但不是总是这样。很可能有些家庭成员间没有因果业报关系。也有可能的是产生一种新的关系，因为这会对所有相关人员的总体情况有利。还有可能的是，一个人有许多不在自己家庭内部的因果业报关系。这些关系可能与一个人在人生轨迹中遇到的人有关，有时候甚至会是很晚才发生的。所有这些（种类的因果关系）背后都有合适而正当的理由和目的。

24.56  QUESTION:  The meaning of the question was this—I did not put it right—is there karmic obligation outside of parental or husband-and-wife relationships?
问题：这个问题的意思是——我刚才没有表达清楚——有没有存在于父母-子女或丈夫-妻子这种关系之外的因果业报方面的义务或责任？

24.57  ANSWER:  There is no such thing as a “karmic obligation.”  The obligation is a law of love and that applies to everyone. There should not be any difference in your attitude whether the relationship is karmic or not. You do not even have to know about it. You have to act according to the laws of God with everyone alike. There is no difference there.
回答：事实上没有所谓的“因果义务”这种说法。这种义务或责任是一个爱心的法则，而这（个法则）适用于所有人。不管这种关系是否有因果的因素，你对它的态度都不应该有任何不同。你甚至不一定需要知道这一点。你需要对每个人做出符合上帝的法则的相同反应。这其中没有区别。

24.58  With this, my friends, I will leave you again. And perhaps next time there will not be quite so many questions, so that I will be able to give at least a short lecture on a topic that I will choose as a beginning for the course.
说到这，我的朋友们，我要再次离开你们了。也许下一次就不会有这么多问题了，这样我至少能就某个我会选为下个系列课程的开端的话题做出一个较短的讲座。

24.59  My friends, I want to tell each one of you, particularly those of you who are troubled, to think of the fact that what you see now is a very blurred outlook, since you are in a heavy and thick cloud. Ask God to give you the strength to penetrate this cloud, to give you at least a small view of truth. But the difficulty is, when you are in this cloud it paralyzes your thinking and your endeavors in this direction. It is often even difficult to muster the strength, the volition to think these thoughts so that you can penetrate, with the help of God, the clouds in which you find yourselves occasionally. But know that you have nothing to fear and nothing to be sad about. Angels of God are forever near every one of God’s children who strive upward with a truly sincere desire. Therefore, rejoice in this knowledge, rejoice upward with a truly sincere desire. Rejoice that God is so much nearer to you, that happiness can be so much nearer than you think if you simply turn around and seek in another direction, if you seek within you instead of without. Be in peace, my dear ones, be blessed, be in God!
我的朋友们，我想告诉你们每个人，特别是那些内心有烦恼的（朋友们），考虑一下这个事实：你们现在看到的是一个非常模糊的外貌，因为你们正处于一个非常沉重、非常厚实的云层里。请求上帝给你穿透这个云层的力量，至少给你一点看到事实真相的空间。但难度在于，当你处在这个云层中时，它会麻痹掉你的思维过程，以及你在这个方向上做出的努力。通常想要鼓起力量、凝聚意志，从而使自己能思考这些问题，进而在上帝的帮助下，穿透这些你们偶尔发现自己所处的云层，这是有一定困难的。不过要知道，你们不需要害怕任何事，也不需要对任何事伤心。上帝的天使永远都在那些抱有真正真诚的渴望向上追求的人们身边。所以，在这份知识中欢欣鼓舞，在真正真诚的向上的渴望中欢欣鼓舞吧。（你们应该）欢欣鼓舞的是上帝离你们更近了，而只要你们简单地改变方向，朝另一个方向追寻，只要你们向内追寻而不是向外追寻，幸福快乐会比你们想象中近得多。愿你们内心平静，我亲爱的朋友们，愿你们受到祝福，与上帝同在！
_
_
©2019 International Pathwork Foundation
Translated by Mark.  Edited by April.
微信订阅号 @Pathwork Lectures 心路讲座中文
